### PR TITLE
Update examples how to use VersionedLayerClient and VolatileLayerClient

### PR DESCRIPTION
**Pull request by AndriyShevchuk**
- Update example how to use VersionedLayerClient
- Update example how to use VolatileLayerClient

Resolves: OLPEDGE-865
Signed-off-by: ashevchu ext-andrii.shevchuk@here.com

### CODE REVIEW

File: docs/examples/nodejs-read-volatile-layer.md [hunk 1]
```diff
@@ -66,15 +66,15 @@ App works!
 ## Log in to the datastore
 
 To work with the datastore, you need to have an [account](https://account.here.com).
-To get your account and create a test app to get app access key id and access key secret, log in to the [HERE Open Location Platform (OLP)](https://platform.here.com/admin/apps) and create a test app.
+To get an access key id and access key secret, log in to the [HERE Open Location Platform (OLP)](https://platform.here.com/admin/apps) and create a test app.
 
-Now, you can get a token for requests to the datastore by using the @here/olp-sdk-authentification.
+Now, you can get a token for requests to the datastore using the @here/olp-sdk-authentification.
```
Comment (HalynaDumych) [inline docs/examples/nodejs-read-volatile-layer.md]:
```suggestion
Now, you can get a token for requests to the datastore using @here/olp-sdk-authentification.
```

File: docs/examples/nodejs-read-volatile-layer.md [hunk 2]
```diff
@@ -66,15 +66,15 @@ App works!
...
 
-Log into the api. Modify our app to the following state:
+Log in to the API. Modify our app to the following state:
 
 ```typescript
 /**
- * Example Node.js app for reading a versioned layer from the datastore
+ * Example Node.js app for reading a volatile layer from the datastore.
```
Comment (HalynaDumych) [inline docs/examples/nodejs-read-volatile-layer.md]:
```suggestion
 * Example Node.js app for reading a volatile layer from the datastore
```

File: docs/examples/nodejs-read-volatile-layer.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ To do so, you need the `OlpClientSettings` object. It contains a download manage
...
```
Comment (HalynaDumych) [inline docs/examples/nodejs-read-volatile-layer.md]:
```suggestion
 * Example Node.js app for reading a volatile layer from the datastore
```

File: docs/examples/nodejs-read-volatile-layer.md [hunk 4]
```diff
@@ -168,9 +168,9 @@ const result = await volatileClient.getData(request);
 The `getPartitions` method expects the following arguments:
 
 * QuadKeyPartitionsRequest `or` PartitionsRequest`.
-* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as a `fetch`) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
+* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as the `fetch` request) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
 
-To fetch partitions metadata from Query API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadKey, depth (from 0 to 4), and version (optional).
+To fetch partitions metadata from the `query` API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadKey, depth (from 0 to 4).
```
Comment (HalynaDumych) [inline docs/examples/nodejs-read-volatile-layer.md]:
```suggestion
To fetch partitions metadata from the `query` API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadkey and depth (from 0 to 4).
```

File: docs/examples/nodejs-read-versioned-layer.md [hunk 5]
```diff
@@ -167,9 +167,9 @@ const result = await versionClient.getData(request);
 The `getPartitions` method expects the following arguments:
 
 * `QuadKeyPartitionsRequest` or `PartitionsRequest`.
-* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as a `fetch`) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
+* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as the `fetch` request) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
 
-To fetch partitions metadata from Query API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadKey, depth (from 0 to 4), and version (optional).
+To fetch partitions metadata from the `query` API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadKey, depth (from 0 to 4), and layer version (optional).
```
Comment (HalynaDumych) [inline docs/examples/nodejs-read-versioned-layer.md]:
```suggestion
To fetch partitions metadata from the `query` API by `QuadKey`, use `QuadKeyPartitionsRequest`. It expects a quadkey, depth (from 0 to 4), and layer version (optional).
```

File: docs/examples/nodejs-read-versioned-layer.md [hunk 6]
```diff
@@ -66,11 +66,11 @@ App works!
 ## Log in to the datastore
 
 To work with the datastore, you need to have an [account](https://account.here.com).
-To get your account and create a test app to get app access key id and access key secret, log in to the [HERE Open Location Platform (OLP)](https://platform.here.com/admin/apps) and create a test app.
+To get an access key id and access key secret, log in to the [HERE Open Location Platform (OLP)](https://platform.here.com/admin/apps) and create a test app.
 
-Now, you can get a token for requests to the datastore by using the @here/olp-sdk-authentification.
+Now, you can get a token for requests to the datastore using the @here/olp-sdk-authentification.
```
[the remark this answers is not in the document]
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-versioned-layer.md]: fixed

File: docs/examples/nodejs-read-versioned-layer.md [hunk 7]
```diff
@@ -119,8 +119,8 @@ const settings = new OlpClientSettings({
 
 ## VersionedLayerClient
 
-When you have  the `OlpClientSettings` object, you can get catalog clients for different catalogs and read the information.
-For information on a version layer following, see [the related section](https://developer.here.com/olp/documentation/get-started/dev_guide/shared_content/topics/olp/concepts/layers.html#versioned-layers) in the Get Started guide.
+When you have  the `OlpClientSettings` object, you can get catalog clients for different catalogs and read information.
+For more information on versioned layers following, see [the related section](https://developer.here.com/olp/documentation/get-started/dev_guide/shared_content/topics/olp/concepts/layers.html#versioned-layers) in the Get Started guide.
```
[the remark this answers is not in the document]
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-versioned-layer.md]: fixed

File: docs/examples/nodejs-read-versioned-layer.md [hunk 8]
```diff
@@ -139,8 +139,8 @@ const versionClient = await new VersionedLayerClient(
 
 The `getData()` method expects the following arguments:
 
-* `dataRequest` &ndash; the `DataRequest` instanse. This class prepares data for the requests to the BlobAPI. The `getData` method can fetch partition data by the following 3 types of parameters you can provide to it. It could be next parameters (sorted by priority): `dataHandle`, `partitionId` and `quadKey`. Below you can take a look on the example of creating `DataRequest` instanse.
-* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as a `fetch`) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
+* `dataRequest` &ndash; the `DataRequest` instanse. This class prepares data for the requests to the BlobAPI. The `getData` method can fetch partition data by the following 3 types of parameters (sorted by priority): `dataHandle`, `partitionId` and `quadKey`. Below, see an example of how to create the `DataRequest` instanse.
```
[the remark this answers is not in the document]
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-versioned-layer.md]: fixed

File: docs/examples/nodejs-read-versioned-layer.md [hunk 9]
```diff
@@ -181,7 +181,7 @@ const request = new QuadKeyPartitionsRequest()
 const result = await versionClient.getPartitions(request);
 ```
 
-To fetch partitions metadata from the MetadataAPI, use `PartitionsRequest`. It expects version parameter (optional). If you skip version and pass an empty `PartitionsRequest` instanse, than the last layer version will be used.
+To fetch partitions metadata from the `metadata` API, use `PartitionsRequest`. It expects version parameter (optional). If you skip version and pass an empty `PartitionsRequest` instanse, than the last layer version will be returned.
```
[the remark this answers is not in the document]
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-versioned-layer.md]: fixed

File: docs/examples/nodejs-read-volatile-layer.md [hunk 10]
```diff
@@ -140,8 +140,8 @@ const volatileClient = await new VolatileLayerClient(
 
 The `getData()` method expects the following arguments:
 
-* `dataRequest` &ndash; the `DataRequest` instanse. This class prepares data for the requests to the BlobAPI. The `getData` method can fetch partition data by the following 3 types of parameters you can provide to it. It could be next parameters (sorted by priority): `dataHandle`, `partitionId` and `quadKey`. Below you can take a look on the example of creating `DataRequest` instanse.
-* `abortSignal` &ndash; a signal object that allows you to communicate with the request (such as a `fetch`) and, if required, abort it  using the `AbortController` object. For more information, see [`AbortSignal` documentation](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal).
+* `dataRequest` &ndash; the `DataRequest` instanse. This class prepares data for the requests to the BlobAPI. The `getData` method can fetch partition data by the following 3 types of parameters (sorted by priority): `dataHandle`, `partitionId` and `quadKey`. Below, see an example of how to create the `DataRequest` instanse.
```
[the remark this answers is not in the document]
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-volatile-layer.md]: fixed

File: docs/examples/nodejs-read-volatile-layer.md (hunk 1 above)
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-volatile-layer.md]: fixed

File: docs/examples/nodejs-read-volatile-layer.md (hunk 2 above)
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-volatile-layer.md]: fixed

File: docs/examples/nodejs-read-volatile-layer.md (hunk 3 above)
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-volatile-layer.md]: fixed

File: docs/examples/nodejs-read-volatile-layer.md (hunk 4 above)
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-volatile-layer.md]: fixed

File: docs/examples/nodejs-read-versioned-layer.md (hunk 5 above)
Comment (AndriyShevchuk) [reply docs/examples/nodejs-read-versioned-layer.md]: fixed

**Pull request merged**